src/js/components/TaskDetailComponent.jsx

```diff
@@ -72,16 +65,16 @@ var TaskDetailComponent = React.createClass({
         {<div>
           <dl className="dl-horizontal">
             <dt>Host</dt>
-            <dd>{task.get("host")}</dd>
+            <dd>{task.host}</dd>
             <dt>Ports</dt>
-            <dd>[{task.get("ports").toString()}]</dd>
+            <dd>[{task.ports.toString()}]</dd>
             <dt>Status</dt>
-            <dd>{task.get("status")}</dd>
+            <dd>{task.status}</dd>
             {timeFields}
             <dt>Version</dt>
             <dd>
-              <time dateTime={task.get("version")}>
-                {task.get("version").toLocaleString()}
+              <time dateTime={task.version}>
+                {new Date(task.version).toLocaleString()}
```

Comment:
I thought that actually instantiating the Date() would fix the issue, but you are right, this is just formatting the retrieved date to the user's locale, not converting the timezone offset.